### ISSUE TITLE
for community implementations v2 or higher, it should use addBeneficiaries instead

### ISSUE THREAD

**Issue by obernardovieira** [impactMarket/utils#145]
https://github.com/impactMarket/impact-market-smart-contracts/blob/main/contracts/community/CommunityImplementation.sol#L576

**Comment by Marianavpt** [comment]
Is this issue solved @obernardovieira and @PauloSousapt? thanks

**Comment by obernardovieira** [comment]
> Is this issue solved @obernardovieira and @PauloSousapt? thanks

No. And currently not in the pipeline. This is an improvement over a mechanism that already exists.
